src/test/java/htsjdk/variant/variantcontext/AlleleUnitTest.java

```diff
@@ -48,7 +47,7 @@
  * Basic unit test for RecalData
  */
 public class AlleleUnitTest extends VariantBaseTest {
```

Comment:
are all the tests in the file unit tests?

Reply:
I'd say that the distinction between unit and non-unit tests in this code base isn't strictly followed (and IMO isn't all that important). It's OK to leave it as-is too, it just seemed odd that it has `Unit` in its name.

Reply:
hmm maybe it was meant to have separate files for different types of tests?  I don't think it would hurt to rename but it seems like theres a bunch of `*UnitTest.java`.  Maybe @droazen has an idea?